supportedBackends/google/pipelines/v2alpha1/src/main/scala/cromwell/backend/google/pipelines/v2alpha1/api/request/AbortRequestHandler.scala

```diff
@@ -4,20 +4,20 @@ import akka.actor.ActorRef
 import com.google.api.client.googleapis.batch.BatchRequest
 import com.google.api.client.googleapis.json.GoogleJsonError
 import com.google.api.client.http.HttpHeaders
+import com.typesafe.scalalogging.LazyLogging
 import cromwell.backend.google.pipelines.common.api.PipelinesApiRequestManager._
-import cromwell.backend.google.pipelines.common.api.clients.PipelinesApiAbortClient.{PAPIAbortRequestSuccessful, PAPIOperationAlreadyCancelled, PAPIOperationHasAlreadyFinished}
+import cromwell.backend.google.pipelines.common.api.clients.PipelinesApiAbortClient.{PAPIAbortRequestSuccessful, PAPIOperationHasAlreadyFinished}
 import cromwell.cloudsupport.gcp.auth.GoogleAuthMode
 
 import scala.concurrent.{ExecutionContext, Future}
 import scala.util.{Failure, Success, Try}
 
-trait AbortRequestHandler { this: RequestHandler =>
+trait AbortRequestHandler extends LazyLogging { this: RequestHandler =>
   protected def handleGoogleError(abortQuery: PAPIAbortRequest, pollingManager: ActorRef, e: GoogleJsonError, responseHeaders: HttpHeaders): Try[Unit] = {
-    // No need to fail the request if the job was already cancelled, we're all good
-    if (Option(e.getCode).contains(400) && Option(e.getMessage).contains("Operation has already been canceled")) {
-      abortQuery.requester ! PAPIOperationAlreadyCancelled(abortQuery.jobId.jobId)
-      Success(())
-    } else if (Option(e.getCode).contains(400) && Option(e.getMessage).contains("Operation has already finished")) {
+    // No need to fail the request if the job already terminated, we're all good
+    // As seen in `cromwell.backend.google.pipelines.common.api.clients.PipelinesApiAbortClient` the difference between "finished" and "cancelled" only affects logging
+    if (Option(e.getCode).contains(400)) {
```

Comment:
I have no love for making behavior decisions based on error message content, so I'm glad that's gone. However, I also worry that a "Bad request" response (I assume this 400 is an HTTP status code) is potentially not specific enough for us to conclude that the job "is no longer running". That said, I have 0 experience with PAPI so... 🤷

Reply:
I'm now thinking maybe we should be checking for the `Precondition failed` string